axlearn/common/flash_attention/neuron_attention_test.py

```diff
@@ -11,6 +11,10 @@
 from axlearn.common.flash_attention.utils import mha_reference
 
 
+if jax.default_backend() != "neuron":
+    pytestmark = pytest.mark.skip(reason="Incompatible hardware, AWS Neuron only test.")
+
```

Comment:
remove blank lines

Reply:
isn't this PEP8? I'll check with linting